### PR TITLE
Part two shout out

### PR DESCRIPTION
**Pull request by karesti**

### CODE REVIEW

File: _posts/2021-06-08-infinispan-redhat-summit-quarkus-part-two.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+tags: [ "redhat", "datagrid", "quarkus", "operator"]
+---
+
+Dear Infinispan Community,
+
+The Infinispan team is pleased to share the follow up article published on the Red Hat Developer blog.
+
+Part two of a two-part series of articles, this blog post focuses on how
+the https://infinispan.org/docs/infinispan-operator/master/operator.html[Infinispan Operator] was used to create
+all the configuration and deployment descriptors for the online hybrid multi cloud Battleship game that featured
```
Comment (oraNod) [inline _posts/2021-06-08-infinispan-redhat-summit-quarkus-part-two.adoc:16]:
```suggestion
all the configuration and deployment descriptors for the online hybrid-cloud Battleship game that featured
```

File: _posts/2021-06-08-infinispan-redhat-summit-quarkus-part-two.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+at this year's Red Hat Summit Keynote.
+
+*You can read the blog post https://developers.redhat.com/articles/2021/06/08/create-and-manage-red-hat-data-grid-services-hybrid-cloud[here]*
+
+The Infinispan Operator is available in the https://operatorhub.io/operator/infinispan[Operator Hub].
```
Comment (oraNod) [inline _posts/2021-06-08-infinispan-redhat-summit-quarkus-part-two.adoc:21]:
```suggestion
The Infinispan Operator is available from the https://operatorhub.io/operator/infinispan[Operator Hub].
```

**Review by oraNod — CHANGES_REQUESTED** [review]

**Comment by karesti** [comment]
@oraNod rebased

**Comment by oraNod** [comment]
merged thanks @karesti

**Pull request merged**